app/frontend/controllers/text_controller.js

```diff
@@ -713,16 +755,33 @@ class TextController {
     }
 
     if (isCache ||
-        listType != 'book' ||
-        listType == 'book' && append ||
-        !isInstantLoadingBook) {
+      listType != 'book' ||
+      listType == 'book' && append ||
+      !isInstantLoadingBook) {
 
       app_controller.optionsMenu.showOrHideSectionTitlesBasedOnOption(tabIndex);
-      await app_controller.initApplicationForVerseList(tabIndex);      
+      await app_controller.initApplicationForVerseList(tabIndex);
       uiHelper.hideTextLoadingIndicator();
     }
   }
 }
 
 module.exports = TextController;
 
+function filterNotesReferenceIds(verseNotes, startVerseNr, endVerseNr) {
```

Comment:
Wouldn't it make more sense to call it `getReferenceIdsFromNotes`?
The current name is a bit ambigous.

Reply:
Yes, thanks for pointing out. I think I started with one function and refactored it into two